### PR TITLE
feat: add ui5 middleware code coverage

### PR DESCRIPTION
**Pull request by flovogt**
Allows client side code coverage generation for ES6+ code powered by https://github.com/SAP/ui5-tooling-extensions/tree/main/packages/middleware-code-coverage.

Projects generated for UI5 versions before 1.113.0 will stick to old "Blanket.js" based code coverage determination limited to ES5 language specification.

JIRA: CPOUI5FOUNDATION-721

### CODE REVIEW

File: generators/app/templates/test/_library_/qunit/testsuite.qunit.js [hunk 1]
```diff
@@ -20,7 +20,11 @@ sap.ui.define(function () {
 				qunitBridge: true,
 				useFakeTimers: false
 			},
-			module: "./{name}.qunit"
+			module: "./{name}.qunit",
+			coverage: {
+				only: ["<%= libURI %>"],
```
Comment (matz3) [inline generators/app/templates/test/_library_/qunit/testsuite.qunit.js:25]:
```suggestion
				only: ["<%= libURI %>/"],
```
ending slash should be added

File: generators/app/templates/test/_library_/qunit/testsuite.qunit.js [hunk 2]
```diff
@@ -20,7 +20,11 @@ sap.ui.define(function () {
...
+				never: ["test-resources/<%= libURI %>"]
```
Comment (matz3) [inline generators/app/templates/test/_library_/qunit/testsuite.qunit.js:26]:
```suggestion
				never: ["test-resources/"]
```
Could be simplified by never instrumenting test-resources modules in general

File: generators/app/templates/test/_library_/qunit/testsuite.qunit.js (hunk 1 above)
Comment (flovogt) [reply generators/app/templates/test/_library_/qunit/testsuite.qunit.js:25]: done

File: generators/app/templates/test/_library_/qunit/testsuite.qunit.js (hunk 2 above)
Comment (flovogt) [reply generators/app/templates/test/_library_/qunit/testsuite.qunit.js:26]: done

**Review by matz3 — APPROVED** [review]

**Pull request merged**